corehq/motech/generic_inbound/views.py

```diff
@@ -0,0 +1,66 @@
+from django.utils.translation import gettext as _
+from psycopg2 import IntegrityError
+
+from corehq.apps.domain.views import BaseProjectSettingsView
+from corehq.apps.hqwebapp.views import CRUDPaginatedViewMixin
+from corehq.motech.generic_inbound.forms import ConfigurableAPIForm
+from corehq.motech.generic_inbound.models import ConfigurableAPI
+
+
+class ConfigurableAPIListView(BaseProjectSettingsView, CRUDPaginatedViewMixin):
+    page_title = _("API Configurations")
```

Comment:
no, but it should be using `gettext_lazy`: 9a806a1efdb

Reply:
I was thinking `noop` because it's explicitly translated later:

https://github.com/dimagi/commcare-hq/blob/f3a33c61d3f739934ca38f6c0e5808ea4382eea6/corehq/tabs/tabclasses.py#L2053

Though I do think `gettext_lazy` is easier to keep track of

Reply:
hmm, good point. I hadn't noticed that. I'll remove the gettext call from tabclasses.

Reply:
[25251f2](https://github.com/dimagi/commcare-hq/pull/32111/commits/25251f28dd7f5a87f099a816c87a2fe00576eba1)